app/preprints/-components/submit/preprint-state-machine/component.ts

```diff
@@ -33,6 +33,8 @@ export default class PreprintStateMachine extends Component<StateMachineArgs>{
     @service store!: Store;
     @service router!: RouterService;
     @service intl!: Intl;
+    titleAndFieldValidation = false;
+    nextButtonIsDisabled = true;
```

Comment:
If you have an object to hold the validation statuses, then this can be a getter like:
```
get nextButtonIsDisabled() {
  return !this.preprintValidationStatuses[this.getTypeFromIndex(this.statusFlowIndex)];
}
```